### PR TITLE
Welcome back Stephanie

### PR DESCRIPTION
**Pull request by chiaberry**
https://github.com/cityofaustin/atd-data-tech/issues/17544

Updated her photo and moved her from alumni to Meet the Team

also fixed the warnings that @frankhereford pointed out, removed the nested p tags and a tags. 

https://deploy-preview-93--atd-product.netlify.app/about

### CODE REVIEW

**Review by amenity — APPROVED** [review]
Thank you, @chiaberry!

**Review by mddilley — APPROVED** [review]
Looks great and thanks for fixing that warning that Frank caught! 🚀 Welcome back, Stephanie! 💻

File: components/shared/IndexIssueListItem.js [hunk 1]
```diff
@@ -34,28 +34,35 @@ export default function IndexIssueListItem({ issue }) {
...
+          <Card.Title className="fw-bold fs-6 text-primary">
+            {issue.title}
+          </Card.Title>
+          <Card.Text className={img ? "card-text-img" : "card-text-no-image"}>
+            <small className="text-muted">
+              <ReactMarkdown
+                components={{
+                  // <Card.Text is a <p> tag, and ReactMarkdown defaults to wrapping its content in a <p> tag
+                  // this throws a nested p tag warning, so changing the p component to fragment get rids of the warning
+                  p: React.Fragment,
```
Comment (johnclary) [inline components/shared/IndexIssueListItem.js:56]: nice! so glad this is finally getting cleaned up 🙏

**Review by johnclary — APPROVED** [review]
🚢 🎉

File: pages/about/index.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ const TEAM_MEMBERS = [
   },
   {
     name: "Mike Dilley",
-    title: "Software Developer",
+    title: "Technical Product Manager",
```
Comment (roseeichelmann) [inline pages/about/index.js:102]: :sparkles:

**Review by roseeichelmann — APPROVED** [review]
lovelyy :heart_eyes_cat:

**Pull request merged**